app/models/user.rb

```diff
@@ -17,6 +24,7 @@ class User < ApplicationRecord
   validates :password, presence: true,
     length: {minimum: Settings.user.length.password_min}, allow_nil: true
 
+  scope :seach_all, ->{order name: :asc}
```

Comment:
```suggestion
  scope :seach_all, ->{order :name}
```
asc là mặc định rồi, xóa dc cho nhẹ ng nhé 

ngoài ra, tên scope đọc nó ko map j với xử lý vậy cu 

đặt 1 cái tên cho nó xịn đến mức ko cần đọc code bên thân vẫn hiểu dc nó là j mới là mục tiêu của chúng ta nhé. em đặt chị đọc còn ko hiểu em muốn j cơ mà.. bớt dương đông kích tây lại 

`search_all` thành `order_name_by_asc"

Reply:
vâng